phpunit/data/themedir1/block-theme/styles/block-style-variation-a.json

```diff
@@ -1,5 +1,6 @@
 {
 	"version": 2,
+	"scope": [ "block" ],
```

Comment:
Is scope the right property here? I'm not sure.

For patterns we use properties like "postTypes", "templateTypes", "inserter"... to indicate the scope. The reason is that we want to be able to provide the exact post types and template types where this shows.

Here it feels like it's very similar. Could this be `"blockTypes": [ "core/group", "core/columns" ] (basically the existing supportedBlockTypes) be enough? or do we want the extra property for something else?

In other words, the presence of "blockTypes" property could be equivalent to scope=block

Reply:
I went with `scope` here for two reasons.

1. To allow a style variation to act as both a block and theme style variation e.g. `"scope": [ "block", "theme" ]`
2. It follows Block Variations which offers the `scope` property

After discussing this with @talldan, I think `scope` is something we could introduce later if we do want the ability to allow a style variation to be used at both the block and theme level.

Also, to clarify, should I be updating `supportedBlockTypes` naming to be `blockTypes`?

Reply:
I've updated this PR to simply check `supportedBlockTypes` for the moment: https://github.com/WordPress/gutenberg/pull/57908/commits/c4f4217beb83d51607621c69cbbe77241c89e283

It's an easy revert if desired. I've also left any renaming of `supportedBlockTypes` until I get confirmation of our desired naming there.

Reply:
> Also, to clarify, should I be updating supportedBlockTypes naming to be blockTypes?

I'd go for blockTypes to match the pattern definitions, but curious about other opinions too.

Reply:
I don't have much of an opinion on this, though I guess as long as it makes sense in the context of other theme.json properties.

Usually those properties are for registering things (e.g. `styles`, `templateParts`), so that could be a point of confusion, though I doubt people will think they can register blocks in the theme.json. 🤔 

It could also be something like `supports.blockTypes` to avoid that, though I don't think it's necessary ... maybe keep it simple.

Reply:
To me, simple is `blockTypes` as it is easier to scan, type etc. 

> Usually those properties are for registering things

That's a fair point although, in a way, this is registering support for given block types. I think the theme.json schema and its description of the property would also help combat any confusion.

It was only `supportedBlockTypes` due to [discussion](https://github.com/WordPress/gutenberg/pull/57787#issuecomment-1898155767) at the time of introducing shared partials for block style variations. 

Ultimately, I don't have any strong opinions on this either. If no alternatives are presented overnight, I'll make the switch tomorrow.

Reply:
I've renamed `supportedBlockTypes` to `blockTypes` in https://github.com/WordPress/gutenberg/pull/57908/commits/a9003929c80a548a78a007d9f01c0453d2d67e9c.